src/engine/api/request.ts

```diff
@@ -9,12 +10,16 @@ export const ACTIVATE_PROJECTS_REQUEST: ProjectRequest = { sourcePath: 'activate
 export const DEPLOY_PROJECTS_REQUEST: ProjectRequest = { sourcePath: 'deploy-projects', description: 'Deploy Ivy Projects' };
 export const CREATE_PROCESS_REQUEST: ProjectRequest = { sourcePath: 'create-process', description: 'Create new Process' };
 export const CREATE_PROJECT_REQUEST: ProjectRequest = { sourcePath: 'create-project', description: 'Create new Project' };
+export const DELETE_PROJECT_REQUEST: ProjectRequest = { sourcePath: 'delete-project', description: 'Delete Project' };
 
 export const PROJECT_REQUEST_OPTIONS: http.RequestOptions = {
   auth: 'Developer:Developer',
   method: 'GET'
 };
 
+export const headers = { 'X-Requested-By': 'web-ide' };
+export const auth: AxiosBasicCredentials = { username: 'Developer', password: 'Developer' };
+
 export function makeGetRequest(url: URL, options: http.RequestOptions): Promise<string> {
```

Comment:
I think I like the idea of having a wrapper around the axios or fetch or whatever lib/api that makes requests and automatically add the correct headers and auth.
Then we woundn't need to export those here and so one...

Reply:
Yes, I think I'll stick to it when I switch to axios completely